packages/components/src/components/option/option.stories.ts

```diff
@@ -34,149 +35,147 @@ export const Default = {
 };
 
 /**
- * Use the `disabled` attribute to disable an option. Clicks will be suppressed until the disabled state is removed. Use the `checkbox` attribute to prefix a styled checkbox. Enabled automatically when using `sd-select` with attribute `checklist` set to `true`.
+ * Use the `variant` attribute to set the option’s variant.
+ * - `default`
+ * - `checkbox`: Enabled automatically when using `sd-select` with attribute `checklist` set to `true`.
+ *
+ * **Hint:**
+ * Option can only be selected when utilized in sd-select.
  */
 
-export const DisabledCheckbox = {
-  name: 'Disabled x Checkbox',
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'disabled',
-          values: [false, true]
-        },
-        y: {
-          type: 'attribute',
-          name: 'checkbox',
-          values: [false, true]
-        }
-      },
-      constants: { type: 'attribute', name: 'disabled', value: true },
-      args
-    });
-  }
+export const Variants = {
+  render: () =>
+    html`<div class="flex flex-row gap-12">
+      <sd-select
+        class="h-[260px] w-[420px]"
+        label="Default"
+        placeholder="Please Select"
+        placement="bottom"
+        max-options-visible="3"
+      >
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+      </sd-select>
+      <sd-select
+        class="h-[160px] w-[420px]"
+        label="Checkbox"
+        placeholder="Please Select"
+        placement="bottom"
+        max-options-visible="3"
+        multiple
+      >
+        <sd-option value="option-1" checkbox>Checkbox Option 1</sd-option>
```

Comment:
Here you can see that the checkbox has no white fill ... that’s missing. Pls open up a bug issue so we can fix it. THX

Reply:
![image](https://github.com/user-attachments/assets/2d9dc309-3b84-4f17-a159-3bdab7c8a38a)

Reply:
https://github.com/orgs/solid-design-system/projects/1/views/1?pane=issue&itemId=78377929